### DISCUSSION TITLE
How to Access Shopify store product data.

### DISCUSSION THREAD

**Discussion by kdmehta2020** [osiset/laravel-shopify#654] [Q&A]
Hello
I have installed This Package and successfully created Basic Shopify App installation and uninstallation but I want to access shopify store data.
is there any way we can [retrieve](url) product data using REST Admin from shopify and store it in application hosting server database? see image
![Shopify APP](https://user-images.githubusercontent.com/12772112/104088700-b88cc900-528e-11eb-9c13-4f9685304842.png)

I know it using client secrete after creating it manually in shopify admin but I am looking for coding help so My App can store all data in Application database on Single Click at application dashboard in app section .

Thanks in advance for help.

**Comment by gnikyt** [comment]
See Usage page of wifi which contains an example code to pull data from shop which you can modify to pull products using a job or something.
